.circleci/config.yml

```diff
@@ -12,15 +12,29 @@ jobs:
       - checkout
       - run:
           name: Validate Terraform configurations
-          command: find . -type f -name "*.tf" -exec dirname {} \;|sort -u | while read m; do (terraform validate -check-variables=false "$m" && echo "√ $m") || exit 1 ; done
+          command: |
+            changed_resources=$(git diff --no-commit-id --name-only -r origin/master...HEAD | grep /resources/ | sort | uniq) || true
+            for _f in ${changed_resources}; do
+              dirname ${_f} | while read m; 
```

Comment:
Is this `while` loop needed?
Would it work if we simply had the following?
```
(terraform validate -check-variables=false "$(dirname ${_f})" && echo "√ $(dirname ${_f})") || exit 1 ;
```

Reply:
Thanks Mitch agree with your suggestion, updated